modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -726,8 +726,17 @@ return view.extend({
 							else
 								exp = '%t'.format(lease.expires);
 
+							var hint = lease.macaddr ? hosts[lease.macaddr] : null,
+							name = hint ? hint.name : null,
```

Comment:
Just one more small comment:
Either indent or `var` at the beginning and with `;` at the end